docs/api/csf.md

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Component Story Format (CSF)'
+---
+
+### Component Story Format (CSF)
+
+Storybook's Component Story Format (CSF) is the recommended way to [write stories](../writing-stories/introduction.md) since Storybook 5.2. [Read the announcement](https://medium.com/storybookjs/component-story-format-66f4c32366df) to learn more about how it came to be.
+
+<div style="background-color:#F8FAFC">
+TODO: vet storiesOf links
+</div>
+
+<div class="aside"
+
+If you are writing stories in the older `storiesOf()` syntax, you can find documentation in an [advanced README](https://github.com/storybookjs/storybook/tree/next/lib/core/ADVANCED.md).
```

Comment:
Can we make these links relative I wonder?

Reply:
converted the link to be relative. Once the file is created it should fall into place.